### PR TITLE
8832 use NR State - CONSUMED, EXPIRED

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#8832

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: src/enums/nr-state.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ export enum NrState {
   COMPLETED = 'COMPLETED', // NOT A VALID STATE
   COND_RESERVED = 'COND-RESERVE', // INTERIM STATE
   CONDITIONAL = 'CONDITIONAL',
+  CONSUMED = 'CONSUMED',
```
Comment (cameron-freshworks) [inline src/enums/nr-state.ts:9]: Is this a state that exists in Namex?

**Review by cameron-freshworks — APPROVED** [review]
Looks good, just 1 question!

File: src/enums/nr-state.ts (hunk 1 above)
Comment (vysakh-menon-aot) [reply src/enums/nr-state.ts:9]: Yes. PR for that in review

Comment (cameron-freshworks) [reply src/enums/nr-state.ts:9]: Very cool, this will be useful in other places as well once Consumed is an official state 👍

**Review by andrepestana-aot — APPROVED** [review]
Looks good!

**Review by severinbeauvais — COMMENTED** [review]
Looks good! (I'm really pleased we get to remove some work-around code!)

Please update the app version so we can easily tell what version is deployed.

File: src/enums/nr-state.ts (hunk 1 above)
Comment (severinbeauvais) [reply src/enums/nr-state.ts:9]: What about Name-Examination UI?

Comment (vysakh-menon-aot) [reply src/enums/nr-state.ts:9]: I am working on that now

**Review by severinbeauvais — APPROVED** [review]

**Pull request merged**